docs/source/noise.rst

```diff
@@ -1162,7 +1163,7 @@ Uncorrupted probability for :math:`\left|000\right\rangle` and :math:`\left|111\
 As expected the outcomes ``000`` and ``111`` each have roughly
 probability :math:`1/2`.
 
-Corrupted probability for :math:`\left|011\right\rangle` and :math:`\left|100\right\rangle`
+Corrupted probability for :math:`\left|000\right\rangle` and :math:`\left|111\right\rangle`
```

Comment:
Also unsure about this one.

Reply:
Link:

http://docs.rigetti.com/en/stable/noise.html#corrupted-probability-for-left-011-right-rangle-and-left-100-right-rangle

Reply:
Looks correct to me.

Reply:
@msohaibalam can you clear up the above.

Reply:
Yeah, I think both "Corrupted probability for ∣∣011⟩ and ∣∣100⟩" and "Corrected probability for ∣∣011⟩ and ∣∣100⟩" following it are typos, they should be "∣000⟩"  and "∣∣111⟩" respectively.